src/abstract/array_with_ids.js

```diff
@@ -1,6 +1,6 @@
-import _ from "underscore";
+import _ from 'underscore';
 
-import {ScalarWithId} from "./scalar_with_id";
+import { ScalarWithId } from './scalar_with_id';
```

Comment:
Not sure about replacing all `"` with `'`. Pycharm/Webstorm by default put double-quotes for imports. Maybe we should just keep it that way?

Reply:
Ok, no problem